lib/src/hydrated.dart

```diff
@@ -122,13 +100,10 @@ class HydratedSubject<T> extends Subject<T> implements ValueStream<T> {
       persist,
       onHydrate,
       subject,
+      persistence,
     );
   }
 
-  /// A unique key that references a storage container
-  /// for a value persisted on the device.
-  String get key => _key;
-
```

Comment:
Could this be considered as breaking change? Do we need to remove it?

Reply:
Restored.